lusSTR/cli.py

```diff
@@ -105,16 +105,55 @@ def snps_subparser(subparsers):
     )
 
 
+def filter_subparser(subparsers):
+    cli = subparsers.add_parser('filter')
+    cli.add_argument(
+        'input',
+        help='Input is a single lusSTR output file (.txt format)'
+    )
+    cli.add_argument(
+        '--separate', action='store_true',
+        help='Used to create separate final output files for each Sample. If not used, a single '
+        'file containing all samples will be created.'
+    )
+    cli.add_argument(
+        '--info', action='store_true',
+        help='Use to create a text document containing additional information on filtered '
+        'sequences and stutter.'
+    )
+    cli.add_argument(
+        '--output-type', dest='output', choices=['efm', 'strmix'], default='efm',
+        help='Choose the file format of the output file, either "efm" or "strmix". '
+        'Default is efm.'
+    )
+    cli.add_argument(
+        '--no-filters', dest='nofilters', action='store_true',
+        help='Used to skip all filtering steps. All input alleles will be included in the output.'
+    )
+    cli.add_argument(
+        '--out', '-o', metavar='FILE',
+        help='Name of output file containing all samples for EFM or name/path of directory for '
+        'STRmix. If separate files are specified for EFM, the sample ID will be used as the '
+        'filename. Output files are in CSV format.'
+    )
+    cli.add_argument(
+        '--profile-type', dest='profile', choices=['evidence', 'reference'], default='evidence',
+        help='Choose the type of profile, either evidence or reference. Default is evidence.'
+    )
```

Comment:
Leaving this question here now, in case I don't find the answer while looking at the rest of the code: how are evidence and reference profiles treated differently? Anything in addition to assuming a single diploid contributor for references?

Reply:
The output format (for both STRmix and EFM) is the only thing that differs (references only report two alleles and do not report the # of reads).